docs/IdentityCard.md

```diff
@@ -0,0 +1,12 @@
+# IdentityCard
+
+- `v::identityCard()`
```

Comment:
*Pesel* may be kept, I think IdentityCard accepting a county code on constructor may be the best option for us.